addon/components/listing.js

```diff
@@ -75,6 +95,12 @@ export default class ListingComponent extends Component {
       ...dataset.triples.map((t) => {
         return { ...t, graph: this.graphs.sourceGraph };
       }),
+      {
+        subject: dataset.sourceNodes[0], // TODO: find out why there can be more than one source node, if so, we should probably insert an order triple for each one?
```

Comment:
This works, I'm not completely sure what this does exactly and in which cases more than one sourceNode would be returned.

Reply:
From memory: there is complexity when you create a node that has inverse relation, e.g. you can have multiple authors attached to the new book you created. This is a case we basically don't support ATM